pages.hi/windows/cmd.md

```diff
@@ -0,0 +1,36 @@
+# cmd
+
+> विंडोज कमांड दुभाषिया।
+> अधिक जानकारी: <https://learn.microsoft.com/windows-server/administration/windows-commands/cmd>।
+
+- एक इंटरैक्टिव शेल सत्र प्रारंभ करें:
+
+`cmd`
+
+- एक आदेश निष्पादित करें:
+
```

Comment:
Same here translate `{{command}}`

Reply:
All the changes have been made as requested. Thanks for the review😊.